src/SignalR/server/Core/src/Internal/DefaultHubDispatcher.cs

```diff
@@ -220,7 +270,10 @@ private Task ProcessStreamItem(HubConnectionContext connection, StreamItemMessag
             THub hub = null;
             try
             {
-                if (!await IsHubMethodAuthorized(scope.ServiceProvider, connection, descriptor.Policies, descriptor.MethodExecutor.MethodInfo.Name, hubMethodInvocationMessage.Arguments))
+                hubActivator = scope.ServiceProvider.GetRequiredService<IHubActivator<THub>>();
+                hub = hubActivator.Create();
+
+                if (!await IsHubMethodAuthorized(scope.ServiceProvider, connection, descriptor, hubMethodInvocationMessage.Arguments, hub))
```

Comment:
We should make sure to call out the ordering of Filters and Authorization in docs. I do remember @rynowak cautioning us against the complexity of moving things like AuthZ **into** filters, so I won't suggest that, but we should at least doc the order.

Reply:
Yah we did that in the old signalr as well but I don’t see the need here

Reply:
Agreed, but we need to doc it :).